cluster-provision/k8s/1.20/nodes.sh

```diff
@@ -5,7 +5,10 @@ set -ex
 source /var/lib/kubevirtci/shared_vars.sh
 
 # Ensure that hugepages are there
-cat /proc/meminfo | sed -e "s/ //g" | grep "HugePages_Total:64"
+if [ -n $HUGEPAGES_COUNT ]; then
```

Comment:
Let's just remove this check. We know that we pass it to the kernel. That should be good enough. If we want to test that, I think we should do it with our conformance tests.